aws-env.dev

```diff
@@ -1,17 +1,21 @@
-export ROOT_URL=https://case.dev.unee-t.com
-export BUGZILLA_ADMIN_KEY=$(aws --profile uneet-dev ssm get-parameters --names BUGZILLA_ADMIN_KEY --with-decryption --query Parameters[0].Value --output text)
-export BUGZILLA_URL=https://dashboard.dev.unee-t.com
-export INVITE_LAMBDA_URL=https://invite.dev.unee-t.com
-export UNIT_CREATE_LAMBDA_URL=https://unit.dev.unee-t.com/create
-export APIENROLL_LAMBDA_URL=https://apienroll.dev.unee-t.com
-export PDFGEN_LAMBDA_URL=https://pdfgen.dev.unee-t.com
-export PDFCONVERT_LAMBDA_URL=https://prince.dev.unee-t.com
-export MONGO_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MONGO_PASSWORD --with-decryption --query Parameters[0].Value --output text)
-export MONGO_CONNECT=$(aws --profile uneet-dev ssm get-parameters --names MONGO_CONNECT --query Parameters[0].Value --output text)
-export CLOUDINARY_PRESET=$(aws --profile uneet-dev ssm get-parameters --names CLOUDINARY_PRESET --with-decryption --query Parameters[0].Value --output text)
-export CLOUDINARY_URL=$(aws --profile uneet-dev ssm get-parameters --names CLOUDINARY_URL --with-decryption --query Parameters[0].Value --output text)
-export API_ACCESS_TOKEN=$(aws --profile uneet-dev ssm get-parameters --names API_ACCESS_TOKEN --with-decryption --query Parameters[0].Value --output text)
-export FROM_EMAIL=$(aws --profile uneet-dev ssm get-parameters --names FROM_EMAIL --query Parameters[0].Value --output text)
-export MAIL_URL=$(aws --profile uneet-dev ssm get-parameters --names MAIL_URL --with-decryption --query Parameters[0].Value --output text)
+getparam () {
+	aws --profile ins-dev ssm get-parameters --names "$1" --with-decryption --query Parameters[0].Value --output text
+}
+export ROOT_URL=https://case.dev.ins.unee-t.com
+export BUGZILLA_ADMIN_KEY=$(getparam BUGZILLA_ADMIN_KEY)
+export BUGZILLA_URL=https://dashboard.dev.ins.unee-t.com
+export INVITE_LAMBDA_URL=https://invite.dev.ins.unee-t.com
+export UNIT_CREATE_LAMBDA_URL=https://unit.dev.ins.unee-t.com/create
+export APIENROLL_LAMBDA_URL=https://apienroll.dev.ins.unee-t.com
+export PDFGEN_LAMBDA_URL=https://pdfgen.dev.ins.unee-t.com
+export PDFCONVERT_LAMBDA_URL=https://prince.dev.ins.unee-t.com
```

Comment:
@kaihendry I'd suggest that we generate these URL based on a rule instead of requesting to manually update these each time.
Something like [sub-component].STAGE.DOMAIN_URL
Example:
ROOT_URL = case.STAGE.DOMAIN
BUGZILLA_URL = dashboard.STAGE.DOMAIN
INVITE_LAMBDA_URL = invite.STAGE.DOMAIN
etc...

Apart from making this simpler, this will also make sure that we have consistent URLs accross all installation and environment for the same things.